### ISSUE TITLE
Tie Concentrations for linear up log down method

### ISSUE THREAD

**Issue by DuyTran16** [billdenney/pknca#46]
How does PKNCA deal with tie concentrations between two consecutive time points when calculating AUC using linear up log down method?

**Comment by billdenney** [comment]
What do you mean by "tie concentrations"?  Do you mean sequential concentrations that are equal like time=1 and time=0 in the data.frame below?

```r
data.frame(time=0:2, conc=c(1, 1, 0.5))
```

If you do mean equal sequential concentrations, linear-trapezoidal is used because the math for logarithmic trapezoidal requires that the earlier concentration is greater than the later concentration (otherwise the math is zero divided by zero times delta-time).

**Comment by DuyTran16** [comment]
I am sorry if I did not provide an example to be more clear.  Yes, I meant equal sequential concentrations, and it makes sense that linear-trapezoidal is used.  I asked this because the results from Phoenix and PKNCA differ for AUC by linear up log down method for this example:
`data.frame(time = c(0, 2, 4, 6, 12), conc = c(50, 100, 100, 48, 65)`

**Comment by DuyTran16** [comment]
After filtering the specific individual PK profile with the AUC that did not match between software when all data were there, it matched afterwards with the filter.  Interesting.

**Comment by billdenney** [comment]
I'm very curious about the difference that you found between PKNCA and WNL.  All NCA software should provide the same answers for the same choices in NCA calculation methods and algorithm choices.  I would appreciate if you could dig a bit more and provide any information on profiles that give different answers between software.

**Comment by DuyTran16** [comment]
I finally figured out the discrepancy.  For some odd reason, the original data had 99.999999999 for the time = 4 and the previous concentration for time = 2 is 100.  I discovered this after exporting the data and viewing this on Excel.  In R, it only display 100 for the concentration at time = 4.  Lesson learned next time is to make sure the concentrations given are rounded up at a specific number of digits before using PKNCA package because the raw data was in Excel and there might be some issues with how the concentration numbers were original given. After fixing this rounding issue, everything matched up between PKNCA and WNL!!!  Thank you so much!
